api/apps/v1alpha1/nimcache_types.go

```diff
@@ -132,6 +132,7 @@ type PersistentVolumeClaim struct {
 	// VolumeAccessMode is the volume access mode of the PVC
 	VolumeAccessMode corev1.PersistentVolumeAccessMode `json:"volumeAccessMode,omitempty"`
 	SubPath          string                            `json:"subPath,omitempty"`
+	ReadOnly         bool                              `json:"readOnly,omitempty"`
```

Comment:
`ReadOnly` should not be part of NIMCache volume spec as we always need it to be rw for caching, instead we need to add it under `NIMService`.

Reply:
I have added `ReadOnly` to `NIMCacheVolSpec`. 
Its here on the PersistenceVolume to support use case in which user has created and populated the PVC without using NIMCache.

Reply:
Fixed after update to the CRD